_posts/2017-02-06-the-4-key-elements-of-a-conversation.md

```diff
@@ -0,0 +1,91 @@
+---
+layout: post
+asset-type: post
+name: The 4 key elements of a conversation
+title: 'The 4 key elements of a conversation'
+date: 2017-02-08 09:00:00 +00:00
+author: Giulia Mantuano
+image:
+  src: /assets/img/custom/blog/2017-02-06-conversational-interface-design-fundamentals.jpg
+tags:
+- UX
+- Chatbot
+- Conversation Design
+- Conversational Interfaces
+
+---
+Conversational Interfaces is a term often used to define systems that hold "human-like" conversations, examples include; Chatbots, Voice User Interfaces, Embodied Conversational Agents. The user interacts with these using text input, voice recognition, and gestures. These types of interfaces are often unable to provide a good conversational experience. In order to create a better experience we need to be aware of the following 4 key elements of human-to-human conversations.
+
+## 1. Turn-taking
+> A turn is one interaction between the user and the system, and a conversation is made of at least two-turns.
+
+<div class="image-side-by-side-text">
+  <img src="/assets/img/custom/blog/2017-02-06-conversational-interface-design-fundamentals/poncho-chatbot.jpg" alt="Hi Poncho on Facebook Messenger" width="250" height="300" />
+</div>
+
+<p class="text-side-by-side-image" >
+<a href="http://poncho.is/">Poncho</a> is a chatbot that provides weather forecasts. Despite having a conversational interface that is better than most, Poncho lets the users down by not providing good turn-taking. As you can see in screenshot, in the second turn the Poncho has to interrupt the conversation because it is not able to understand the user’s prompt. That is not a good conversation. In this example, the conversation only has one turn. In a real human-to-human conversation that’s similar to meeting a friend and having the following dialog:</p>
+
+<pre>
+Me: “Hi Jane, How are you?”
+Jane: “Hi, I’m good thank you!”
+<emp>Jane walks away...</emp>
+</pre>
+
+<p>
+Now that's an awkward conversation –people rarely have conversations that last a single turn.
```

Comment:
Now that's an awkward conversation – people rarely have conversations that last only for a single turn.